code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm

```diff
@@ -364,4 +364,4 @@
 		src << "<span class='warning'>You miss!</span>"
 		return
 
-	T.Weaken(3)
+	T << "<span class='warning'>Please lay down for the arrest to commence immersively.</span>"
```

Comment:
This is just silly now.

I didn't say we had to delete the pounce feature, just that it cannot be accepted when it's as overpowered as it is. Consider what mobs are _normally_ able to do, such as using disarm to knock someone down.

Reply:
Alright. What I can tell is that disarm has a 25% chance to push, which will apply the same 3 second weaken if the target has <60 armor. I'm just having difficulties trying to figure out how to make the chances feasible for it to not require too many costly attempts while still not being an insta stun. Could possibly take down the cooldown and just rely on the charge costs so that an rng-unlucky borg will hop its batteries dry.